temporalcli/commands.go

```diff
@@ -495,6 +504,14 @@ func (c *TemporalCommand) preRun(cctx *CommandContext) error {
 		}
 	}
 	cctx.JSONShorthandPayloads = !c.NoJsonShorthandPayloads
+
+	if c.RpcTimeout.Duration() != time.Second*10 {
+		// We multiply the timeout by 2 because the client SDK sets the RPC timeout
+		// to half of the Deadline
+		newDeadline := time.Now().Add(c.RpcTimeout.Duration() * 2)
```

Comment:
This is why I was afraid of calling it "RPC timeout". There are many CLI commands that make multiple RPC calls. Doubling the user timeout makes it no longer accurate to what the user supplied. The reason the Go SDK halves it is to retry, but i am not sure we need to affect the entire context that governs timeout for the whole CLI command with that.

Maybe "context timeout" is an acceptable term, or "command timeout" or something.